psql2mysql.py

```diff
@@ -61,6 +61,37 @@ def _exclude_deleted(self, table, query):
             return query.where(table.c.deleted == 'False')
         return query.where(table.c.deleted == False)
 
+    def getTextColumns(self, table):
+        columns = table.columns
+        return [ c.name for c in columns if str(c.type) == 'TEXT' ]
+
+    def scanTableForLongTexts(self, table):
+        textColumns = self.getTextColumns(table)
+        if not textColumns:
+          return []
+        LOG.debug("Scanning Table %s (columns: %s) for too long TEXT values ",
+                 table.name, textColumns)
+        filters = [
+            text("length(\"%s\") > %i" % (x, MAX_TEXT_LEN)) for x in textColumns
+        ]
+        q = table.select().where(or_(f for f in filters))
+        rows = q.execute()
+
+        long_values = []
+        primary_keys = []
+        if not table.primary_key == None:
+          primary_keys = list(table.primary_key)
+        for row in rows:
+            for col in textColumns:
+                if not isinstance(row[col], six.string_types):
+                    continue
```

Comment:
oh can this happens? be a text column but do not contain text?

Reply:
It can actually be None, in which case the next call fails. Comes from https://github.com/rhafer/psql2mysql/commit/07fcb15425365123a80f085248cc9e385f550e76